README.md

```diff
@@ -46,7 +46,7 @@ Check out [the documentation](https://backstage.io/docs/getting-started) on how
 ## Community
 
 - [Discord chatroom](https://discord.gg/MUpMjP2) - Get support or discuss the project
-- [Good First Issues](https://github.com/backstage/backstage/contribute) - Start here if you want to contribute
+- [Contributing to Backstage](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md) - Start here if you want to contribute
```

Comment:
The contribution guidelines should be the starting point, and they eventually lead to good first issues as well, but contains more useful information.